### PR TITLE
feat: limited build to k8s >= 1.20.x

### PR DESCRIPTION
**Pull request by csviri**

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17 ]
-        kubernetes: [ 'v1.17.17','v1.18.20','v1.19.16','v1.20.15','v1.21.12', 'v1.22.9', 'v1.23.6', 'v1.24.0' ]
+        kubernetes: ['v1.20.15','v1.21.12', 'v1.22.9', 'v1.23.6', 'v1.24.0' ]
         exclude:
           - java: 11
             kubernetes: 'v1.18.20'
```
Comment (metacosm) [inline .github/workflows/pr.yml:39]: You also need to remove these excludes that are not needed anymore…

Comment (csviri) [reply .github/workflows/pr.yml]: ahh, yes, thx. done.

Comment (csviri) [reply .github/workflows/pr.yml]: btw, its not clear to me why exactly those are excluded, what is the rational behind those values? I mean why are those exactly excluded. 
Should be there just the oldest and the newest not excluded for java 11? @metacosm

Comment (metacosm) [reply .github/workflows/pr.yml]: I guess the idea was to test only the latest versions with java 11 to limit the CI intensity.

**Review by metacosm — APPROVED** [review]

**Pull request merged**